src/btree/row_modify.c

```diff
@@ -435,7 +435,8 @@ __wt_update_obsolete_check(
          * moved forwards. This function is used to trim update lists independently of the page
          * state, ensure there is a modify structure.
```

Comment:
@amiyuki7, please can you remove the last part of the sentence: `, ensure there is a modify structure`? We already checked earlier line 354 with the `WT_PAGE_TRYLOCK` macro.

Reply:
Yep, semantic fix has been pushed.